dts/arm/nordic/nrf52833.dtsi

```diff
@@ -0,0 +1,374 @@
+			status = "okay";
+			label = "CLOCK";
+		};
+
+		uart0: uart@40002000 {
+			/* uart can be either UART or UARTE, for the user to pick */
+			/* compatible = "nordic,nrf-uarte" or "nordic,nrf-uart"; */
+			reg = <0x40002000 0x1000>;
+			interrupts = <2 1>;
+			status = "disabled";
+			label = "UART_0";
+		};
+
+		uart1: uart@40028000 {
+			compatible = "nordic,nrf-uarte";
+			reg = <0x40028000 0x1000>;
+			interrupts = <40 1>;
+			status = "disabled";
+			label = "UART_1";
+		};
+
+		gpiote: gpiote@40006000 {
+			compatible = "nordic,nrf-gpiote";
+			reg = <0x40006000 0x1000>;
+			interrupts = <6 5>;
+			status = "disabled";
+			label = "GPIOTE_0";
+		};
+
+		gpio0: gpio@50000000 {
+			compatible = "nordic,nrf-gpio";
+			gpio-controller;
+			reg = <0x50000000 0x200
+			       0x50000500 0x300>;
+			#gpio-cells = <2>;
+			label = "GPIO_0";
+			status = "disabled";
+		};
+
+		gpio1: gpio@50000300 {
+			compatible = "nordic,nrf-gpio";
+			gpio-controller;
+			reg = <0x50000300 0x200
+			       0x50000800 0x300>;
+			#gpio-cells = <2>;
+			label = "GPIO_1";
+			status = "disabled";
+		};
+
+		i2c0: i2c@40003000 {
+			/*
+			 * This i2c node can be TWI, TWIM, or TWIS,
+			 * for the user to pick:
+			 * compatible = "nordic,nrf-twi" or
+			 *              "nordic,nrf-twim" or
+			 *              "nordic,nrf-twis".
+			 */
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40003000 0x1000>;
+			clock-frequency = <I2C_BITRATE_STANDARD>;
+			interrupts = <3 1>;
+			status = "disabled";
+			label = "I2C_0";
+		};
+
+		i2c1: i2c@40004000 {
+			/*
+			 * This i2c node can be TWI, TWIM, or TWIS,
+			 * for the user to pick:
+			 * compatible = "nordic,nrf-twi" or
+			 *              "nordic,nrf-twim" or
+			 *              "nordic,nrf-twis".
+			 */
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40004000 0x1000>;
+			clock-frequency = <I2C_BITRATE_STANDARD>;
+			interrupts = <4 1>;
+			status = "disabled";
+			label = "I2C_1";
+		};
+
+		pwm0: pwm@4001c000 {
+			compatible = "nordic,nrf-pwm";
+			reg = <0x4001c000 0x1000>;
+			interrupts = <28 1>;
+			status = "disabled";
+			label = "PWM_0";
+			#pwm-cells = <1>;
+		};
+
+		pwm1: pwm@40021000 {
+			compatible = "nordic,nrf-pwm";
+			reg = <0x40021000 0x1000>;
+			interrupts = <33 1>;
+			status = "disabled";
+			label = "PWM_1";
+			#pwm-cells = <1>;
+		};
+
+		pwm2: pwm@40022000 {
+			compatible = "nordic,nrf-pwm";
+			reg = <0x40022000 0x1000>;
+			interrupts = <34 1>;
+			status = "disabled";
+			label = "PWM_2";
+			#pwm-cells = <1>;
+		};
+
+		pwm3: pwm@4002d000 {
+			compatible = "nordic,nrf-pwm";
+			reg = <0x4002d000 0x1000>;
+			interrupts = <45 1>;
+			status = "disabled";
+			label = "PWM_3";
+			#pwm-cells = <1>;
+		};
+
+		qdec: qdec@40012000 {
+			compatible = "nordic,nrf-qdec";
+			reg = <0x40012000 0x1000>;
+			interrupts = <18 1>;
+			status = "disabled";
+			label = "QDEC";
+		};
+
+		spi0: spi@40003000 {
+			compatible = "nordic,nrf-spi";
```

Comment:
This should not be set here. Force the user to select the proper type, just like in [nrf52840.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/nordic/nrf52840.dtsi#L217). Same goes for `spi1` and `spi2`.